frontend/src/styles/Expenses.css

```diff
@@ -0,0 +1,35 @@
+.expense{
+    height:100vh;
+    display: grid;
+    grid-template-rows: 2fr 9fr 1fr;
+    /* max-width: 500px; */
+    margin: 0 auto;
+    
+}
+
+.bcg{
+    background-color: var(--primary-bg-color);
+}
+
+.expenseform{
+    display: grid;
+    height: 25vh;
+    grid-template-columns: 2fr 2fr 1fr;
+    grid-gap: 1rem;
+    padding: 1rem;
+}
+
+button{
+    height: 38px !important;
+    background-color: var(--secondary-color) !important;
+    border: none !important;
+}
+
+.expPres{
+    display: grid;
+    grid-template-columns: 3fr 9fr;
+    grid-gap: 1rem;
+    padding: 1rem;
+    padding-top: 2rem;
+    text-align: center;
+}
```

Comment:
These should be moved to ExpensePresentation.css

"display:flex" and "justify-content:space-between" would probably work better here than grid/grid-template/grid-gap/text-align

Font styling is missing.

Reply:
1. Moved to seperate file.

2. Changed to display:flex.
Font Styling changed, but had to eyeball it, as design does not give specific numbers, Or it does, but I dint quite understood.

Reply:
Resolved in 98ae7b35ef0c274798064fc2b47af099f5e6cfa2